config/notifiers.go

```diff
@@ -101,14 +101,8 @@ var (
 		NotifierConfig: NotifierConfig{
 			VSendResolved: true,
 		},
-		Message:   `{{ template "wechat.default.message" . }}`,
-		APIURL:    `{{ template "wechat.default.api_url" . }}`,
-		APISecret: `{{ template "wechat.default.api_secret" . }}`,
-		ToUser:    `{{ template "wechat.default.to_user" . }}`,
-		ToParty:   `{{ template "wechat.default.to_party" . }}`,
-		ToTag:     `{{ template "wechat.default.to_tag" . }}`,
-		AgentID:   `{{ template "wechat.default.agent_id" . }}`,
-		// TODO: Add a details field with all the alerts.
+		Message: `{{ template "wechat.default.message" . }}`,
+		APIURL:  DefaultGlobalConfig.WeChatAPIURL,
```

Comment:
It would be good to leave this as a templated value for users. There is the check in https://github.com/prometheus/alertmanager/pull/1229/files#diff-3baf47c64847a8fb8aaa8cc2e088513bR250 that will set the api url to the global default.

Reply:
ok, I will check it again.